corehq/motech/fhir/tests/test_utils.py

```diff
@@ -163,12 +163,27 @@ class TestHeaderDecorators(TestCase):
     def setUp(self):
         self.factory = RequestFactory()
 
-    def _do_get_request(self, view, headers={}):
-        request = self.factory.get('/foo', **headers)
+    def _do_get_request(self, view, params=None, headers={}):
+        request = self.factory.get('/foo', params, **headers)
         return view(request)
 
-    def test_require_fhir_json_accept_headers(self):
-        @require_fhir_json_accept_headers
+    def test_validate_format_param(self):
+        @validate_accept_header_and_format_param
+        def test_view(request):
+            return HttpResponse()
+        response = self._do_get_request(test_view)
+        self.assertEqual(response.status_code, 200)
+        response = self._do_get_request(test_view, {'_format': 'application/json'})
+        self.assertEqual(response.status_code, 200)
+        response = self._do_get_request(test_view, {'_format': 'application/xml'})
+        self.assertEqual(response.status_code, 406)
+        self.assertEqual(
+            response.content.decode("utf-8"),
+            '{"message": "Requested format not acceptable."}'
+        )
+
+    def test_validate_accept_header(self):
+        @validate_accept_header_and_format_param
```

Comment:
To be more complete it might be good to use POST for either this or the previous test.